marketplace/charts/marketplace-integration.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM gcr.io/cloud-builders/gcloud-slim
```

Comment:
I don't see this file used?

I remember there was a Dockerfile that is used for only side effects of publishing the chart to the GCS bucket. Is this the same one? We want to remove that too, right?

Reply:
You're right - this is that file. It was a mistake to include - removing.